SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -12446,7 +12446,12 @@ void CEulerSolver::TurboAverageProcess(CSolver **solver, CGeometry *geometry, CC
                   Radius2    = geometry->GetTurboRadius(iMarker,iSpan);
                   Radius1    = geometry->GetTurboRadius(iMarker,iSpan-1);
                   Vt2        = AverageTurboVelocity[iMarker][iSpan - 1][1]*AverageTurboVelocity[iMarker][iSpan - 1][1];
-                  RadialEquilibriumPressure[iMarker][iSpan -1] =  RadialEquilibriumPressure[iMarker][iSpan] - AverageDensity[iMarker][iSpan -1]*Vt2/Radius1*(Radius2 - Radius1);
+                  if (Radius1 < 1E-15){
+                  RadialEquilibriumPressure[iMarker][iSpan -1] =  RadialEquilibriumPressure[iMarker][iSpan] - AverageDensity[iMarker][iSpan -1]*Vt2/Radius2*(Radius2 - Radius1);
+                  }
+                  else{
+                    RadialEquilibriumPressure[iMarker][iSpan -1] =  RadialEquilibriumPressure[iMarker][iSpan] - AverageDensity[iMarker][iSpan -1]*Vt2/Radius1*(Radius2 - Radius1);
+                  }
```

Comment:
c'mon... not like this...
Avoid if's inside inner loops, not good for performance, please don't repeat code, creates bugs, and don't use "magic numbers" (1e-15) there is an EPS defined in option_structure.hpp that can be used for this kind of generic check.
```
...
Radius1 = geometry->GetTurboRadius(iMarker,iSpan-1);
Radius1 = (Radius1 > EPS)? Radius1 : Radius2;
...
```
Sorry for asking for changes after approving.

Reply:
lol I knew you suggested something like that's why I didn't merge it. I didn't think of this syntax so I went the crude way. I fixed it now.